src/js/components/project-view/setup-instructions.tsx

```diff
@@ -1,98 +1,79 @@
   text: string;
-  'smaller-text': string;
   code: string;
   url: string;
 }
 
 interface Props {
   project: Project;
-  styles?: Styles;
-  hideLabels?: boolean;
-  showClone?: boolean;
 }
 
-const selectText = (e: React.MouseEvent<HTMLElement>) => {
-  const node = e.target;
-  const doc = document as any;
-
-  if (doc.selection) {
-    const range = doc.body.createTextRange();
-    range.moveToElementText(node);
-    range.select();
-  } else if (window.getSelection) {
-    const range = document.createRange();
-    range.selectNodeContents(node as Node);
-    window.getSelection().removeAllRanges();
-    window.getSelection().addRange(range);
-  }
-};
-
-const SetupInstructions = ({ project, hideLabels, showClone, styles: passedInStyles }: Props) => {
-  const styles = passedInStyles || defaultStyles;
+const SetupInstructions = ({ project }: Props) => {
   return (
     <div className={styles.instructions}>
       <div className={styles.section}>
-        {!hideLabels && (
-          <div className={styles.label}>
-            Git repository
-          </div>
-        )}
+        <div className={styles.label}>
+          Code repository
+        </div>
         <div className={styles.text}>
           The URL for this project's Git repository is
           <div className={styles.url}>{project.repoUrl}</div>
         </div>
-        {showClone && (
-          <div>
-            <div className={styles.text}>
-              Clone the repo with
-            </div>
-            <div className={styles.code}>
-              <pre onClick={selectText}>git clone -o minard {project.repoUrl}</pre>
-            </div>
+
+        <div>
+          <div className={styles.text}>
+            Clone the repository with
+          </div>
+          <div className={styles.code}>
+            <pre onClick={selectText}>git clone -o minard {project.repoUrl}</pre>
           </div>
-        )}
+        </div>
+
         <div className={styles.text}>
-          {showClone ?
-            'Or you can add it as a remote to your existing repository with the following command.' :
-            'Add it as a remote to your existing repository with the following command and start pushing some code.'
-          }
+          …Or connect your existing project folder with
         </div>
         <div className={styles.code}>
-          <pre onClick={selectText}>git remote add minard {project.repoUrl}</pre>
+          <pre onClick={selectText}>
+            git init<br />
+            git add .<br />
+            git commit -m “First commit”<br />
+            git remote add origin {project.repoUrl}<br />
+            git push -u origin master
+          </pre>
         </div>
+
         <div className={styles.text}>
-          If you use another repository (such as GitHub) as your origin, set up Git to automatically
-          also push to Minard with the following command.
+          …Or, if you already use Git in your project, add Minard as a new remote with
```

Comment:
Same thing as above: already using Git in your project is not the same as having an existing `origin` (i.e. remote repository).